tests/conftest.py

```diff
@@ -18,6 +18,12 @@
 PLATFORM = platform_key()
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "no_mocking_on_ci: Do not mock locations when running on the CI."
```

Comment:
The rest of the `conda` repos have this kind of config in `pyproject.toml`. It might be better to follow that trend for consistency.

Reply:
Ah, good to know. I wasn't sure since I didn't see any pytest-related parts in `pyproject.toml`. Before I implements this, I'm wondering if we should generally disable mocking on the CI, but that might be out of scope.